### PR TITLE
Chagatai with JtF bug

### PR DESCRIPTION
**Pull request by The-Outcast**
Chagatai should not keep standing when attacking own provinces with JtF's effect.

### CODE REVIEW

File: server/game/cards/07-WotW/MotoChagatai.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ class MotoChagatai extends DrawCard {
         this.persistentEffect({
             condition: context => (
                 context.source.isAttacking() &&
+                context.game.currentConflict.conflictProvince.controller !== context.source.controller &&
```
Comment (marcsoiferman) [inline server/game/cards/07-WotW/MotoChagatai.js:16]: I don't think this is the right fix

Comment (marcsoiferman) [reply server/game/cards/07-WotW/MotoChagatai.js:16]: I think the better way is to update onBreakProvince to track which players have had a province broken this conflict, and then check if your current opponent has lost a province.  This should take into account a bunch of weird scenarios with Stoke, Raze to the Ground, taking control effects, etc.

**Review by marcsoiferman — CHANGES_REQUESTED** [review]

File: server/game/cards/07-WotW/MotoChagatai.js (hunk 1 above)
Comment (The-Outcast) [reply server/game/cards/07-WotW/MotoChagatai.js:16]: This takes into account Stoke and control effects. I have to check what Raze does.

Comment (marcsoiferman) [reply server/game/cards/07-WotW/MotoChagatai.js:16]: We can add the Raze condition as part of implementing raze.

**Review by marcsoiferman — APPROVED** [review]

**Pull request merged**